anvil/src/eth/api.rs

```diff
@@ -1672,7 +1672,7 @@ impl EthApi {
                     lowest_gas_limit = mid_gas_limit;
                 }
                 reason => {
-                    trace!(target: "node", "estimation failed due to {:?}", reason);
+                    warn!(target: "node", "estimation failed due to {:?}", reason);
```

Comment:
Should these be errors instead of warnings?

Reply:
these are not necessarily errors that affect anvil itself, but essentially failed transactions, I think `warn` is appropriate here.

Reply:
Hmm Ok I can see that.